src/libsystemd/sd-bus/bus-internal.h

```diff
@@ -256,6 +256,7 @@ typedef struct sd_bus {
 
         int *fds;
         size_t n_fds;
+        bool fds_might_be_truncated; /* MSG_CTRUNC was seen during recvmsg() */
```

Comment:
i wuld really prefer if this would just be called got_ctrunc or so, i.e. say what actually happened.